Fix grammar (#2078)
flyteorg/flyte: rsts/concepts/admin.rst

Admin Structure
===============

> FlyteAdmin serves the main Flyte API to process all client requests to the system. Clients include the FlyteConsole, that calls:

Reviewer: ```suggestion
Flyte Admin serves as the main Flyte API to process all client requests to the system. Clients include Flyte Console, which calls:
```

> 1. FlyteAdmin to list the workflows, get execution details, etc.,;
> 2. FlyteKit to call FlyteAdmin to register workflows, launch workflows, etc.

Reviewer: ```suggestion
1. FlyteAdmin to list the workflows, get execution details, etc.
2. Flytekit that in turn calls FlyteAdmin to register, launch workflows, etc.
```

Common
------

> As the name implies, ``common`` houses shared components used across different FlyteAdmin components in a single, top-level directory to avoid cyclic dependencies. These components include execution naming and phase utils, query filter definitions, query sorting definitions, and named constants.

Reviewer: ```suggestion
As the name implies, ``common`` houses shared components used across different Flyte Admin components in a single, top-level directory to avoid cyclic dependencies. These components include execution naming and phase utils, query filter definitions, query sorting definitions, and named constants.
```

Runtime
-------

> Values specific to the FlyteAdmin application, including task, workflow registration, and execution are configured in the `runtime <https://github.com/flyteorg/flyteadmin/tree/master/pkg/runtime>`__ directory. These interfaces expose values configured in the ``flyteadmin`` top-level key in the application config.

Reviewer: ```suggestion
Values specific to the Flyte Admin application, including task, workflow registration, and execution are configured in the `runtime <https://github.com/flyteorg/flyteadmin/tree/master/pkg/runtime>`__ directory. These interfaces expose values configured in the ``flyteadmin`` top-level key in the application config.
```

Entities
---------

The :std:ref:`admin service definition <protos/docs/service/service:flyteidl/service/admin.proto>` defines REST operations for the entities that

> FlyteAdmin administers.

Reviewer: ```suggestion
Flyte Admin administers.
```

> As a refresher, the primary :ref:`entities <divedeep>` across Flyte maps to FlyteAdmin entities.

Reviewer: ```suggestion
As a refresher, the primary :ref:`entities <divedeep>` across Flyte maps to Flyte Admin entities.
```

- Tasks
- Launch Plans

Permitted operations include:

- Create
- Get
- List

The above entities are designated by an :std:ref:`identifier <protos/docs/core/core:identifier>`
that consists of a project, domain, name, and version specification. These entities are, for the most part, immutable. To update one of these entities, the updated
version must be re-registered with a unique and new version identifier attribute.

> One caveat is that the launch plan can toggle between :std:ref:`ACTIVE or INACTIVE <protos/docs/admin/admin:launchplan>` states.

Reviewer: ```suggestion
One caveat is that the launch plan can toggle between :std:ref:`ACTIVE and INACTIVE <protos/docs/admin/admin:launchplan>` states.
```

At a given point in time, only one launch plan version across a shared project, domain and name specification can be active. The state affects the scheduled launch plans only.

> An inactive launch plan can also be used to launch individual executions. However, only an active launch plan runs on a schedule (given it has a schedule defined).

Reviewer: ```suggestion
An inactive launch plan can be used to launch individual executions. However, only an active launch plan runs on a schedule (given it has a schedule defined).
```

Static entities metadata (Named Entities)
+++++++++++++++++++++++++++++++++++++++++

A :std:ref:`named entity <protos/docs/admin/admin:namedentity>` includes metadata for one of the above entities
(workflow, task or launch plan) across versions. It also includes a resource type (workflow, task or launch plan) and an
:std:ref:`id <protos/docs/admin/admin:namedentityidentifier>` which is composed of project, domain and name.
The named entity also includes metadata, which are mutable attributes about the referenced entity.

This metadata includes:

- Description: a human-readable description for the Named Entity collection.

> - State (workflows only): this determines whether the workflow is shown on the overview list of workflows scoped by project and domain.

Reviewer: I don't think we need periods at the end of the bulleted list items.

  - created_at	
  - updated_at	
  - workflows.{any workflow field above} (for example: workflow.domain)	

>   - state (you must use the integer enum ex: 1)	

Reviewer: ```suggestion
  - state (you must use the integer enum, e.g., 1)	
```

     - States are defined in :std:ref:`launchplanstate <protos/docs/admin/admin:launchplanstate>`.
     
- Named Entity Metadata

>   - state (you must use the integer enum ex: 1)	

Reviewer: ```suggestion
  - state (you must use the integer enum, e.g., 1)	
```

  - name	
  - workflow.{any workflow field above} (for example: workflow.domain)	
  - launch_plan.{any launch plan field above} (for example: launch_plan.name)	

>   - phase (you must use the upper-cased string name ex: ``RUNNING``)	

Reviewer: same as above ^ .e.g.,

Reviewer: Please use this format wherever you used `ex:`.

Putting It All Together	
-----------------------	

> If you wish to query specific executions that were launched using a specific launch plan for a workflow with specific attributes, it would look similar to:

Reviewer: ```suggestion
If you wish to query specific executions that were launched using a specific launch plan for a workflow with specific attributes, you could something like:
```

Sorting syntax	
--------------	

Adding sorting to a request requires specifying the ``key``, ex: the attribute you wish to sort on. Sorting can also optionally specify the direction (one of ``ASCENDING`` or ``DESCENDING``) where ``DESCENDING`` is the default.	

Example sorting HTTP parameter:	

::	

   sort_by.key=created_at&sort_by.direction=DESCENDING	
   	

> Alternatively, since `DESCENDING` is the default sorting direction, the above could be written as	

Reviewer 2: this is rst unf not markdown.  in rst if you want to `monospace` something you have to use two backticks
```
``mono``
```
